src/components/NcEllipsisedOption/NcEllipsisedOption.vue

```diff
@@ -20,6 +20,54 @@
   -
   -->
 
+<docs>
+A component to display a long text with highlight support in one line truncated with ellipsis in the end but keeping up to 10 last characters.
+
+It is supposed to be used as an `NcSelect`'s option in first place.
+
+```vue
+<template>
+	<div>
+		<h5>Plain text</h5>
```

Comment:
should be h4 no?

Reply:
Looks too huge to me 👀

But can replace it with `h4`

Reply:
Or with separated examples

Reply:
I'll do it

Reply:
Done